api/tests/Feature/Notifications/TriggerNewJobPostedTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Tests\Feature\Notifications;
+
+use App\Models\Pool;
+use App\Models\Team;
+use App\Models\User;
+use App\Notifications\NewJobPosted;
+use Database\Seeders\RolePermissionSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Notification;
+use Nuwave\Lighthouse\Testing\RefreshesSchemaCache;
+use Tests\TestCase;
+
+class TriggerNewJobPostedTest extends TestCase
+{
+    use RefreshDatabase;
+    use RefreshesSchemaCache;
+
+    private User $adminUser;
+
+    private User $regularUser;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Notification::fake();
+
+        $this->seed(RolePermissionSeeder::class);
+        Team::factory()->create();
+
+        $this->adminUser = User::factory()
+            ->asApplicant()
+            ->asAdmin()
+            ->create([
+                'sub' => 'adminUser',
+                'ignored_email_notifications' => [],
+                'ignored_in_app_notifications' => [],
+            ]);
+        $this->regularUser = User::factory()
+            ->asApplicant()
+            ->create([
+                'sub' => 'regularUser',
+                'ignored_email_notifications' => [],
+                'ignored_in_app_notifications' => [],
+            ]);
+    }
+
+    // no notification when an application is created or submitted
+    public function testNothingSentForDraftPool(): void
+    {
```

Comment:
Here

Reply:
Fixed in [fix test comments](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10358/commits/7cc36f9056382efd23c36ad65d7f0fec10ea8ce5)